lib_src/doc/rst/lib_src.rst

```diff
@@ -529,6 +534,87 @@ ASRC Processing
 
 .. doxygenfunction:: asrc_process
 
+Fixed factor of 3 functions
+===========================
+
+The SRC library also includes synchronous sample rate conversion functions to downsample (decimate) and oversample (upsample or interpolate) by a fixed factor of three. In each case, the processing is carried out each time a single output sample is required. In the case of the decimator, three input samples passed to filter with a resulting one sample output on calling the processing function. The interpolator produces an output sample each time the processing function is called but will require a single sample to be pushed into the filter every third cycle. All samples use Q31 format (left justified signed 32b integer).
+
+Both sample rate converters are based on a 144 tap FIR filter with two sets of coefficients available, depending on application requirements:
+
+ * firos3_b_144.dat / firds3_b_144.dat - These filters have 20dB of attenuation at the nyquist frequency and a higher cutoff frequency
+ * firos3_144.dat / firds3_144.dat - These filters have 60dB of attenuation at the nyquist frequency but trade this off with a lower cutoff frequency
+
+The filter coefficients may be selected by adjusting the line::
+
+  #define   FIROS3_COEFS_FILE
+
+and::
+
+  #define   FIRDS3_COEFS_FILE
+
+in the files ``os3.h`` (API for oversampling) and ``ds3.h`` (API for downsampling) respectively.
+
+The OS3 processing takes up to 153 core cycles to compute a sample which translates to 1.53us at 100MHz or 2.448us at 62.5MHz core speed. This permits up to 8 channels of 16KHz -> 48KHz sample rate conversion in a single 62.5MHz core.
+
+The DS3 processing takes up to 389 core cycles to compute a sample which translates to 3.89us at 100MHz or 6.224us at 62.5MHz core speed. This permits up to 9 channels of 48KHz -> 16KHz sample rate conversion in a single 62.5MHz core.
```

Comment:
(1/16000)/6.24e-6 - 10.016, so we can squeeze 10 in theoretically.
My mistake - I should have re-calculated this bit